sdv/single_table/copulagan.py

```diff
@@ -135,6 +149,7 @@ def __init__(self, metadata, enforce_min_max_values=True, enforce_rounding=True,
             cuda=cuda,
         )
 
+        self._validate_numerical_distributions(numerical_distributions)
```

Comment:
This can be changed for `GaussianCopulaSynthesizer._validate_numerical_distributions(numerical_distributions, self.metadata._columns)`